src/utils/toSQL.js

```diff
@@ -397,6 +397,71 @@ export function jsonToSQLite(obj) {
     .join("\n");
 }
 
+export function jsonToDjangoModels(obj) {
+  let djangoCode = "";
```

Comment:
Please add "from django.db import models\n" to the 1st line

Reply:
Resolved